src/Riok.Mapperly/Diagnostics/DiagnosticDescriptors.cs

```diff
@@ -213,4 +213,20 @@ internal static class DiagnosticDescriptors
         DiagnosticCategories.Mapper,
         DiagnosticSeverity.Info,
         true);
+
+    public static readonly DiagnosticDescriptor MultipleConfigurationsForConstructorArgument = new DiagnosticDescriptor(
```

Comment:
I think the term parameter would fit better here than argument (same in the message and for RMG028).